### ISSUE TITLE
Investigate why tests on Mac are taking some much time dependent on slave

### ISSUE THREAD

**Issue by vpakan** [jboss-reddeer/reddeer#1174]
[INFO] Red Deer Eclipse Component Tests .................. SUCCESS [2:03:38.397s] - zamek
[INFO] Red Deer Eclipse Component Tests .................. SUCCESS [1:12:04.270s] - domek

**Comment by mmalina** [comment]
I checked this today and it's still the case:
[INFO] Red Deer Eclipse Component Tests .................. SUCCESS [45:27.104s] - domek
[INFO] Red Deer Eclipse Component Tests .................. SUCCESS [1:00:04.965s] - domek
[INFO] Red Deer Eclipse Component Tests .................. SUCCESS [1:25:49.997s] - zamek

I was puzzled as to why there's such a big difference. Both are Mac minis, zamek is even newer, so should be faster if anything. There are SSD drives and 8 GB of RAM on both machines.

But then Vlado asked if the OS is the same, and it's not.
domek - 10.10 Yosemite
zamek - 10.9 Mavericks

So I will update zamek to El Capitan and see if it helps.

**Comment by mmalina** [comment]
I upgraded zamek to 10.11 El Capitan and it's still just as slow as before. So there must be some other reason. I will try to compare disk speeds and then run some benchmarks.

**Comment by mmalina** [comment]
I found out that the Samsung SSD disks on both machines were very slow due to disabled TRIM (OS X does not use it for third party disks by default). After enabling it, the disks are fast again.
Then I ran Geekbench 3 and it shows that zamek is faster, which is expected.
But dummy job still takes less than 3 minutes on domek and more than 5 minutes on zamek :(
So I need to dig deeper.
Some ideas: Could it be something with the network? Or is there firewall running on one of the machines? I will check later.
